ibm/resource_ibm_kms_key.go

```diff
@@ -36,6 +36,13 @@ func resourceIBMKmskey() *schema.Resource {
 				ForceNew:    true,
 				Description: "Key protect or hpcs instance GUID",
 			},
+			"key_ring_id": {
```

Comment:
value of key_ring_id is not set in the read method..

Reply:
Docs Update for resource is missing

Reply:
Its set with d.Set("key_ring_id") line 185